### PR TITLE
Use pdns_server to obtain the PowerDNS version

### PR DESCRIPTION
**Pull request by atosatto**
Removes the dependency from `atosatto.packages-extras` and uses the `pdns_server` command to get the PowerDNS version.

### CODE REVIEW

**Comment by atosatto** [comment]
@pieterlexis, I will update the outdated README shortly.

File: tasks/inspect.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+- name: Obtain the PowerDNS version
+  shell: "pdns_server --version 2>&1 | awk '/PowerDNS Authoritative/{print $7}'"
+  register: pdns_version_output
+
+- debug:
+    msg: "{{ pdns_version_output }}"
+
+- set_fact:
+    pdns_package_version: "{{ pdns_version_output['stdout'] }}"
```
Comment (pieterlexis) [inline tasks/inspect.yml]: does this need an `rstrip()`?

**Comment by pieterlexis** [comment]
Can you test this with pre-releases, like the 4.1 repo we have now?

The binary will report `4.1.0-rc1`, but apt will know it as `4.1.0~rc1-1pdns.stretch` and for yum this is `4.1.0-0.1.rc1.1pdns.el7`

File: tasks/inspect.yml (hunk 1 above)
Comment (atosatto) [reply tasks/inspect.yml]: Why?

**Comment by atosatto** [comment]
I am writing molecule v2 tests to be added to this PR to double check all that it's working fine in most of the different scenarios.

File: tasks/inspect.yml (hunk 1 above)
Comment (pieterlexis) [reply tasks/inspect.yml]: stdout might contain a `\n`

Comment (atosatto) [reply tasks/inspect.yml]: Yes. But it's not the case of the string we're using to extract the version from with `awk`. Column `$7` is not the last one :)

**Comment by atosatto** [comment]
@pieterlexis just pushed the tests. Is it possible to have Travis running for this repo?

**Comment by atosatto** [comment]
@lieter, the PR should be reviewed :) No more `WIP` prefix.

**Comment by atosatto** [comment]
Just pushed some improvements to the README trying to make more clear the usage of `pdns_package_version` and addressing @jpmens's feedback regarding the MySQL backend (see #40).

**Pull request merged**